index.html

```diff
@@ -138,16 +138,37 @@ <h2 class="pa-h2-v2-hero-subhead pa-white">Let us do the technological and profe
     <!-- /.pa-hero -->
 
 
-    <!-- CTA Section
-	================================================== -->
-    <section class="pa-cta-v1" data-parallax="scroll" data-speed="0.2" data-image-src="assets/images/image-section.jpg">
+     
+	<section class="pa-cta-v1" data-parallax="scroll" data-speed="0.2" data-image-src="assets/images/web5.jpg">
 
-        <div class="container">
-            <h1 class="pa-h1-v1 pa-bright fadeInLeftDelay03Duration10"><strong>Coolest HTML Theme Yet?</strong></h1>
-            <h4 class="pa-h4-v1 pa-bright fadeInLeftDelay03Duration10">Open Particle today, why not!</h4>
-            <div class="mt-4"></div>
-            <a class="pa-button-primary pa-hvr-sweep-to-right fadeInRightDelay11Duration10" href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">Get it Now</a>
+    <div class="text-center">
+    <div class="row">
+        <div class="col-lg-12">
+            <h3 style="font-size:60px"> About Us </h3>
+        </div>
+    </div>
+    </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6">
+                <div class="container">
+                   <h1> We make our <br>people and products<br>sensational.</h1>
+                            <br>
+                            <p>To revolutionize the way people do software and delivering happiness to our stakeholders and society.</p>
+                            <br>
+                        </div>
+                        <a href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">Schedule Your Free Consultation &nbsp;</a>
```

Comment:
please make it to a proper button like 
![image](https://user-images.githubusercontent.com/29180114/125891088-8449c723-fdc3-438e-b530-f75bf797337c.png)

Reply:
done